lxd/state/state.go

```diff
@@ -30,11 +31,14 @@ type State struct {
 	// Event server
 	DevlxdEvents *events.Server
 	Events       *events.Server
+
+	// Firewall instance
+	Firewall firewall.Firewall
```

Comment:
Can you move that change and related changes to its own commit?
That would include:
 - Changes to lxd/state/state.go
 - Changes to lxd/daemon.go
 - Changes to lxd/state/testing.go